build/circle-test.sh

```diff
@@ -77,8 +77,18 @@ prepare_artifacts() {
         -d "${2}"
       }
 
+      echo "Posting an issue"
       # Generate string of failed tests: 'TestRaftRemoveRace TestChaos TestHoneyBooBoo'
-      FAILEDTESTS=$(grep -oh '^--- FAIL: \w*' "${outdir}/excerpt.txt" | sed -e 's/--- FAIL: //' | tr '\n' ' ')
+      FAILEDTESTS=$(grep -oh '^--- FAIL: \w*' "${outdir}/excerpt.txt" | sed -e 's/--- FAIL: //' | tr '\n' ' ' || true)
```

Comment:
That's a good question. I don't think we do (at least `go2xunit` obviously catches dirty failures, so it'll catch all clean ones for sure).